packages/components/src/components/input-field/input.jsx

```diff
@@ -65,10 +66,11 @@ const Input = ({
                         className={classNames(
                             classNameInlinePrefix ? `${classNameInlinePrefix}--symbol` : '',
                             'symbols',
-                            `symbols--${inline_prefix.toLowerCase()}`,
                             { disabled: !!is_disabled }
                         )}
-                    />
+                    >
+                        {inline_prefix === 'UST' ? getCurrencyDisplayCode(inline_prefix) : inline_prefix}
```

Comment:
Should we write a function for this in `@deriv-shared`'s `currency.js` to centralise these special cases?

Reply:
`{getCurrencyDisplayCode(inline_prefix)}` should be enough. This will also yield the same result. We don't need to check for a special case here.